src/main/java/TestFile.java

```diff
@@ -32,4 +32,11 @@ private static void onMoreVariationOfIncorrectComparingToItself() {
             // whatever
         }
     }
+
+    private static void inappropriateIndentation() {
```

Comment:
[Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.